### PR TITLE
Add docs for distortion

### PR DESCRIPTION
**Pull request by Flova**

### CODE REVIEW

File: soccer.rst [hunk 1]
```diff
@@ -111,7 +112,7 @@ and the default dimensions of the objects.
 They also include the ``footpoint_out_of_image_threshold`` parameter which is used to determine whether the footpoint of an object is out of the image or not.
 Objects with a footpoint below this fraction of the image height are considered to be out of the image.
 If the real footpoint of an object is not visible, it is ignored as using the lowest point of the object as its footpoint will result in a wrong projection.
-
+The ``use_distortion`` parameter is used to determine whether to use the distortion parameters from the camera info or not. Don't use them if you detect the objects in an rectified image.
```
Comment (jaagut) [inline soccer.rst]:
```suggestion
The ``use_distortion`` parameter is used to determine whether to use the distortion parameters from the camera info or not. Don't use them if you detect the objects in an already rectified image.
```

File: tutorial.rst [hunk 2]
```diff
@@ -194,7 +199,9 @@ Please read the comments in the code to understand what is needed for the projec
             # Initialize the IPM library with a reference to the forward kinematics of the robot
             # We also need to provide the camera info, this is optional during the initialization
             # as it can be provided via a setter later on as well
-            self.ipm = IPM(self.tf_buffer, self.camera_info)
+            # We can also set the distortion parameter to True if we want to use the distortion model 
+            # from the camera info and don't want to rectify the image beforehand
+            self.ipm = IPM(self.tf_buffer, self.camera_info, distortion=True)
```
Comment (jaagut) [inline tutorial.rst:204]: It is True

File: tutorial.rst [hunk 3]
```diff
@@ -348,7 +357,9 @@ In this case, a NumPy array instead of the Point2D ROS message is used as our in
             # Initialize the IPM library with a reference to the forward kinematics of the robot
             # We also need to provide the camera info, this is optional during the initialization
             # as it can be provided via a setter later on as well
-            self.ipm = IPM(self.tf_buffer, self.camera_info)
+            # We can also set the distortion parameter to True if we want to use the distortion model
+            # from the camera info and don't want to rectify the image beforehand
+            self.ipm = IPM(self.tf_buffer, self.camera_info, distortion=True)
```
Comment (jaagut) [inline tutorial.rst:362]: Same

**Review by jaagut — APPROVED** [review]

File: tutorial.rst (hunk 2 above)
Comment (Flova) [reply tutorial.rst:204]: I agree that the wording is a bit weird, but I don't think the comment necessarily contradicts the fact that it is true in the example.

**Pull request merged**